bioptim/examples/getting_started/example_implicit_dynamics.py

```diff
@@ -0,0 +1,197 @@
+import matplotlib.pyplot as plt
+import numpy as np
+
+
+def prepare_ocp(
+    biorbd_model_path: str,
+    final_time: float,
+    n_shooting: int,
+    ode_solver: OdeSolver = OdeSolver.RK1(n_integration_steps=1),
+    use_sx: bool = False,
+    n_threads: int = 1,
+    implicit_dynamics: bool = False,
+) -> OptimalControlProgram:
+    """
+    The initialization of an ocp
+
+    Parameters
+    ----------
+    biorbd_model_path: str
+        The path to the biorbd model
+    final_time: float
+        The time in second required to perform the task
+    n_shooting: int
+        The number of shooting points to define int the direct multiple shooting program
+    ode_solver: OdeSolver = OdeSolver.RK4()
+        Which type of OdeSolver to use
+    use_sx: bool
+        If the SX variable should be used instead of MX (can be extensive on RAM)
+    n_threads: int
+        The number of threads to use in the paralleling (1 = no parallel computing)
+    implicit_dynamics: bool
+        implicit
+    Returns
+    -------
+    The OptimalControlProgram ready to be solved
+    """
+
+    biorbd_model = biorbd.Model(biorbd_model_path)
+
+    objective_functions = ObjectiveList()
+    objective_functions.add(ObjectiveFcn.Lagrange.MINIMIZE_CONTROL, key="tau")
+
+    # Dynamics
+    dynamics = Dynamics(DynamicsFcn.TORQUE_DRIVEN, implicit_dynamics=implicit_dynamics, implicit_soft_contacts=False)
+
+    # Path constraint
+    tau_min, tau_max, tau_init = -100, 100, 0
+
+    # Be careful to let the accelerations not to much bounded to find the same solution in implicit dynamics
+    if implicit_dynamics:
+        qddot_min, qddot_max, qddot_init = -20000, 20000, 0
+
+    x_bounds = BoundsList()
+    x_bounds.add(bounds=QAndQDotBounds(biorbd_model))
+    x_bounds[0][:, [0, -1]] = 0
+    x_bounds[0][1, -1] = 3.14
+
+    # Initial guess
+    n_q = biorbd_model.nbQ()
+    n_qdot = biorbd_model.nbQdot()
+    n_tau = biorbd_model.nbGeneralizedTorque()
+    x_init = InitialGuess([0] * (n_q + n_qdot))
+
+    # Define control path constraint
+    # There are extra controls in implicit dynamics which are joint acceleration qddot.
+    if implicit_dynamics:
+        u_bounds = Bounds([tau_min] * n_tau + [qddot_min] * n_tau, [tau_max] * n_tau + [qddot_max] * n_tau)
+    else:
+        u_bounds = Bounds([tau_min] * n_tau, [tau_max] * n_tau)
+
+    u_bounds[n_tau - 1, :] = 0
+
+    if implicit_dynamics:
+        u_init = InitialGuess([0] * n_q * 2)
+    else:
+        u_init = InitialGuess([0] * n_q)
+
+    return OptimalControlProgram(
+        biorbd_model,
+        dynamics,
+        n_shooting,
+        final_time,
+        x_init=x_init,
+        u_init=u_init,
+        x_bounds=x_bounds,
+        u_bounds=u_bounds,
+        objective_functions=objective_functions,
+        ode_solver=ode_solver,
+        use_sx=use_sx,
+        n_threads=n_threads,
+    )
+
+
+def main():
+    """
+    If pendulum is run as a script, it will perform the optimization and animates it
```

Comment:
Can you update this to explain that you are creating two OCP and then plot them together